Dockerfile

```diff
@@ -7,7 +7,7 @@ ADD . /go/src/github.com/cosmo0920/fluent-bit-go-s3
 WORKDIR /go/src/github.com/cosmo0920/fluent-bit-go-s3
 RUN make build
 
-FROM fluent/fluent-bit:1.3
```

Comment:
fluent/fluent-bit:1.3 always points to v1.3 latest.
If you want to rebuild per the latest fluent-bit 1.3 series, how about the following?

```
LABEL Description="Fluent Bit Go S3" FluentBitVersion="1.3.9"
```

Reply:
We could add the label too indeed, specifying the version in the `FROM` ensure we are effectively using the right version.

Reply:
Make sense.